gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/DatastoreReader.java

```diff
@@ -45,13 +45,13 @@
 
   /**
    * Returns a list with a value for each given key (ordered by input).
-   * A {@code null} would be returned for non-existing keys.
-   * When possible prefer using {@link #get(Key...)} which does not eagerly loads the results.
+   * {@code null} values are returned for nonexistent keys.
+   * When possible prefer using {@link #get(Key...)} to avoid eagerly loading the results.
```

Comment:
Nit: I think this should either be on the above line or in a new paragraph (Also applies to the above line)